components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -20,7 +20,7 @@ info:
     Please see [full OpenAPI Specification](https://raw.githubusercontent.com/wso2/carbon-apimgt/master/components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml) of the API which is written using [OAS 3.0](http://swagger.io/) specification.
 
     # Authentication
-    The Admin REST API is protected using OAuth2 and access control is achieved through scopes. Before you start invoking
+    The Admin REST APIs are protected using OAuth2 and access control is achieved through scopes. Before you start invoking
```

Comment:
There is only one API with multiple resources right?

Reply:
Yes. This is fixed now.